README.md

```diff
@@ -9,31 +9,35 @@
 ## Getting Started
 1. Import library features you will use:
 
-```
+```gradle
 implementation 'com.github.getkevin.kevin-android:core:xxx'
 implementation 'com.github.getkevin.kevin-android:accounts:xxx'
 implementation 'com.github.getkevin.kevin-android:in-app-payments:xxx'
 ```
 or import all features all together:
-```
+```gradle
 implementation 'com.github.getkevin:kevin-android:xxx'
 ```
 2. Initialize plugins you will use in your Application file:
 
-```
-Kevin.addPlugin(KevinAccountsPlugin)
-Kevin.addPlugin(KevinPaymentsPlugin)
-Kevin.configure(
-    KevinConfiguration
-        .builder()
-        .setTheme(R.style.KevinTheme)
+```kotlin
+Kevin.setLocale(Locale("en"))   //  optional locale setup (by default phone locale will be used)
+Kevin.setTheme(R.style.KevinTheme)  //  supply optional theme
```

Comment:
Theme is not optional at this moment. Without it, sdk will crash. Probably we could add Theme.Kevin.Base as default theme in config, but then we should also create night theme

Reply:
typo